README.md

```diff
@@ -41,7 +41,9 @@ For more details, see [Plugin Management](https://docs.fluentd.org/v0.14/article
   * Default value: no
 * **remove_tag_prefix** (string) (optional): Remove tag prefix for tag placeholder. (see the section of "Tag placeholder")
 * **hostname_command** (string) (optional): Override hostname command for placeholder. (see the section of "Tag placeholder")
-  * Default value: `hostname`.
+  * Default value: `hostname`
+* **emit_mode** (enum) (required): Specify emit_mode to `record` or `batch`. `record` will emit events per record. `batch` will emit events per rewritten tag.
```

Comment:
Writing merit is better for `batch`.
I think default should be `batch` since v2.0 because `batch` mode is good for popular in_tail / in_forward's case

Reply:
Do you mean that default will be `batch` in next release? (The current version of this plugin is v2.1.1)
I will release v2.2.0 after merging this PR.

Reply:
Ah, sorry, 2.2.0. If you have a concern, 3.0 is ok for me.

Reply:
I got it. Thanks!